website/src/documentation/how-tos.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: section
+title: "How Tos"
+section_menu: section-menu/documentation.html
+permalink: /documentation/how-tos/
+redirect_from:
+  - /learn/how-tos/
```

Comment:
don't need the redirect lines, that was just for old pages before we had our new hierarchy

Reply:
Got it, thanks